### DISCUSSION TITLE
When will I get my benefits after I have been approved?

### DISCUSSION THREAD

**Discussion by SaiffMoh** [community/community#123550] [GitHub Education] [answered]
### Hi everyone,

Hi everyone,
so the thing is I have been approved and it is saying that:
![Capture](https://github.com/community/community/assets/127222866/6d298475-80f2-4a6d-a2a7-d55934b76a4a)

I am just curious how long it takes till I get the benefits. Any answer is appreciated!

**Comment by davevad93** [comment, marked as answer]
Hi @SaiffMoh ,

The time it takes may vary depending on different factors, mine for example was accepted in 2 days, I've heard of people who had to wait 9-10 days and others more, but in general it takes 2-3 days. When your benefits are fully available it shows the following message: 

`Congrats! Your request for @SaiffMoh was approved and your benefits are now available.`
